components/FeatureHighlight/src/MDCFeatureHighlightViewController.m

```diff
@@ -169,15 +169,26 @@ - (void)viewWillTransitionToSize:(CGSize)size
        withTransitionCoordinator:(id<UIViewControllerTransitionCoordinator>)coordinator {
   [super viewWillTransitionToSize:size withTransitionCoordinator:coordinator];
   UIViewController *presenter = self.presentingViewController;
-  UIViewController *presentingViewController = self;
+  UIViewController *presentedViewController = self;
   [self dismissViewControllerAnimated:NO completion:nil];
```

Comment:
I'm not sure I follow.
Do we want to dismiss then immediately present self to re-display the feature highlight based on the new size?
Why aren't we just moving the center of the featureHightlightView?

Reply:
Hmm, so the person who reported the bug thinks that it was introduced in e91de06. That commit appears to replace code that sounds _similar_ to what you're describing with this current dismiss/present again strategy. I was assuming that was done for a good reason, but it's possible it wasn't.

Reply:
The background on that change, if I recall correctly, was that there were animation glitches caused by attempting to animate the frame (i.e. the frame of the "source view" changes).

That being said, I agree with @ianegordon that it's likely in our best interest long-term to explore whether we can properly re-animate the frame.